script.js

```diff
@@ -1,8 +1,90 @@
+/* eslint-disable max-classes-per-file */
+
+let books = JSON.parse(localStorage.getItem('books')) || [];
+
+// |||book class
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// |||user interface interactions
+class UserInterface {
+  static container = document.getElementById('container');
+
+  static titleInput = document.querySelector('#title');
+
+  static authorInput = document.querySelector('#author');
+
+  static addButton = document.querySelector('.add-book');
+
+  static addBook() {
+    const book = new Book(
+      UserInterface.titleInput.value,
+      UserInterface.authorInput.value,
+    );
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+
+    UserInterface.titleInput.value = '';
+    UserInterface.authorInput.value = '';
+    UserInterface.displayBook(book, books.length - 1);
+  }
```

Comment:
- [ ] The behavior of your app can be optimized to function better. Currently, when a book is added it is persisted to the `localStorage`, which is good. However, the page needs to be reloaded for the book to appear on the page. It would be nice if this is not the case. This can be resolved by modifying line 33 and appending the result to the page like so:
  
  ```
  const bookContainer = UserInterface.displayBook(book, books.length - 1);
  UserInterface.container.appendChild(bookContainer)
  ```
  This way when a book is created it is added to the page and also persisted on the `localStorage`